core/src/main/scala/org/http4s/QValue.scala

```diff
@@ -144,11 +148,44 @@ object QValue {
   @deprecated("""use qValue"" string interpolation instead""", "0.20")
   def q(d: Double): QValue = macro Macros.qValueLiteral
 
-  implicit val http4sOrderForQValue: Order[QValue] = Order.fromOrdering[QValue]
-  implicit val http4sShowForQValue: Show[QValue] = Show.fromToString[QValue]
-  implicit val http4sHttpCodecForQValue: HttpCodec[QValue] = new HttpCodec[QValue] {
-    def parse(s: String): ParseResult[QValue] = QValue.parse(s)
-    def render(writer: Writer, q: QValue): writer.type = q.render(writer)
+  implicit val catsInstancesForHttp4sQValue: Order[QValue]
```

Comment:
Combining these is good, but it's going to break binary compatibility on version 0.21.  We should keep the old instances, make them not implicit, and deprecate them for version 0.21.20.  To avoid code duplication, I think they can just be aliases to this new combined instance.

Reply:
understood. Thank you for your comment.
I fixed it.